packages/gasket-plugin-nextjs/lib/setup-next-app.js

```diff
@@ -18,14 +18,13 @@ function getPortFallback(env = '') {
  * @private
  */
 async function setupNextApp(gasket) {
-  const { exec, command, config } = gasket;
+  const { exec, config } = gasket;
   const { hostname, http, https, http2, env } = config;
   const createNextApp = require('next');
-  const devServer = (command.id || command) === 'local';
+  const devServer = Boolean(process.env.GASKET_DEV); // TODO document GASKET_DEV
```

Comment:
Is `GASKET_DEV` going to be official then? I don't have a better suggestion at the moment.

Reply:
I think so I also don't have a better alternative. It's simple and explicit.

Reply:
Documentation should be covered in our docs tickets